src/Search.js

```diff
@@ -48,16 +48,16 @@ class Search extends Component {
                     </Col>
```

Comment:
Please just make sure to add some comments to your code describing what each section does!